data/tlg1269/tlg002/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1269" xml:lang="grc" urn="urn:cts:greekLit:tlg1269.tlg002">
+    <ti:title xml:lang="lat">Cleanthis Assii Fragmenta et Apophthegmata.</ti:title>
+    <ti:edition urn="urn:cts:greekLit:tlg1269.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg1269.tlg002">
+        <ti:label xml:lang="lat">Cleanthis Assii Fragmenta et Apophthegmata.</ti:label>
+        <ti:description xml:lang="mul">Cleanthes, Fragmenta and Apophthegmata, von Arnim, Teubner, 1903</ti:description>
```

Comment:
description should be 
**Common author name.** **Book title.** **Volume (if applicable).** **Editor Last, Editor First,** editor. **City: Publisher, Year.**

<ti:description xml:lang="mul">Cleanthes. Stoicorum veterum fragmenta, Vol. 1. von Arnim, Hans Friedrich August, editor. Stuttgart: Teubner, 1903. (1964 printing.)</ti:description>